src/chunk.c

```diff
@@ -2606,6 +2606,7 @@ chunk_tuple_delete(TupleInfo *ti, DropBehavior behavior, bool preserve_chunk_cat
 
 		Assert(!form.dropped);
 
+		form.compressed_chunk_id = INVALID_CHUNK_ID;
```

Comment:
compressed_chunk_id should be marked as NULL.

Reply:
INVALID_CHUNK_ID will turn into database NULL on write (in src/chunk.c)